### ISSUE TITLE
Translation to Spanish

### ISSUE THREAD

**Issue by UlisesTJ** [cloudfoundry/cf-deployment#415]
hello I would like to collaborate with your project by serving as a translator to Spanish

**Comment by knz** [comment]
@UlisesTJ https://github.com/cockroachdb/cockroach/issues/23038#issuecomment-368220870

**Comment by dsabeti** [comment]
Thanks for chiming in, @knz. Your comment in that repo is very well thought out and captures the general feedback I was hoping to give as well.

@UlisesTJ (cc @LuigiBaute), I appreciate that you're interested in helping out. The Cloud Foundry foundation doesn't have a project or budget for translating documentation in our repos to other languages. If you're interested in helping out for free, I'd advise sending an email to cf-dev@lists.cloudfoundry.org to see if there's a need for this in the CF community.

**Comment by UlisesTJ** [comment]
Of course, my contribution will be free, just look for them to merge my translations @dsabeti

**Comment by UlisesTJ** [comment]
?? @dsabeti

**Comment by dsabeti** [comment]
Hi @UlisesTJ, did you send an email to cf-dev@lists.cloudfoundry.org? I think it would be best to get some feedback and input from the larger community.

**Comment by UlisesTJ** [comment]
The translation would be for free of course, what do I have to email to @cf-dev ? @dsabeti

**Comment by dsabeti** [comment]
Oh, even if you're interested in helping out for free, it would be good to get input and help from the larger community. In particular, I think that's the place to start a discussion about starting a translation project.
